CHANGES.rst

```diff
@@ -1,12 +1,19 @@
 Changelog
 =========
 
-0.6.1 (unreleased)
-==================
+2.0 (unreleased)
```

Comment:
It might be good to add a comment here, where release 1.0.0 went. [... was an internal release ...]

Reply:
Okay.